packages/core/src/common/tetherUtils.ts

```diff
@@ -14,6 +14,19 @@ const DEFAULT_CONSTRAINTS = {
     to: "scrollParent",
 };
 
+// per https://github.com/HubSpot/tether/pull/204,
+// Tether now exposes a `bodyElement` option that,
+// when present, gets the tethered element injected
+// into *it* instead of into the document body. but
+// both approaches still cause React to freak out,
+// because it loses its handle on the DOM element.
+// thus, we pass a fake HTML bodyElement to Tether,
+// with a no-op `appendChild` function (the only
+// function the library uses from bodyElement.)
+const fauxHtmlElement = ({
```

Comment:
Please avoid using "faux" in the terminology and instead opt for a more straightforward name, `fakeBodyElement`. It's harder to follow for non-native English speakers. -1 for this